packages/wasm/crate/src/view_server.rs

```diff
@@ -58,11 +57,9 @@ impl ScanBlockResult {
 #[wasm_bindgen]
 pub struct ViewServer {
     latest_height: u64,
-    epoch_duration: u64,
     fvk: FullViewingKey,
     notes: BTreeMap<note::StateCommitment, SpendableNoteRecord>,
     swaps: BTreeMap<tct::StateCommitment, SwapRecord>,
-    denoms: BTreeMap<Id, Metadata>,
```

Comment:
regarding the `epoch_duration` field, this [discord comment](https://discord.com/channels/824484045370818580/979785365437165658/1062518567968448583) seems relevant. 

When comparing the `ViewService` protobuf definitions, I noticed that the  [epoch_index](https://buf.build/penumbra-zone/penumbra/docs/main:penumbra.view.v1#penumbra.view.v1.TransactionPlannerRequest) in the `TransactionPlannerRequest` is now deprecated. It seems that at some point, it may have been necessary for transaction planning before being deprecated possibly?

I traced the earlier view server wasm implementation back to last year, but I didn't find any useful comments on why these fields were introduced.